### PR TITLE
Celestia Node v0.5.0

### PR DESCRIPTION
**Pull request by renaynay**
https://github.com/celestiaorg/celestia-node/commit/4667f42f9f823ee95e5c85d593d5a3972e4c6fbc breaks node flags such that `--node.network` is now `--p2p.network` and should be specified on both `init` and `start`

Resolves part of https://github.com/celestiaorg/celestia-node/issues/1271

### CODE REVIEW

**Comment by YazzyYaz** [comment]
Hi @renaynay is this new flag in v0.4.1?

**Comment by renaynay** [comment]
No @YazzyYaz, but it will be for next release so should we leave this PR open? Maybe we can have some sort of labels on docs

**Comment by YazzyYaz** [comment]
Ok, let's move this PR to Draft and label it for 0.4.2 or 0.5.0, whichever is the new release. All our docs point to 0.4.1 atm

Upon release, we can merge this PR. 

Also, we will be removing the `i18n` directory in this PR https://github.com/celestiaorg/docs/pull/271 so we might need to update this PR after `271` is merged to not add the `i18n` files.

File: docs/developers/node-tutorial.mdx [hunk 1]
```diff
@@ -488,10 +488,10 @@ version` command:
...
 ```output
 celestia version
-Semantic version: v0.4.2
-Commit: 75b122f736d0db758ef840a4bf3b2e6635e823bd
-Build Date: Thu Oct 20 22:48:39 EDT 2022
-System version: amd64/darwin
+Semantic version: v0.5.0-rc5
+Commit: 039facd94104c741f8fa995e80f1321776ae4ad3
+Build Date: Tue Nov 15 16:23:30 EST 2022
+System version: arm64/darwin
```
Comment (jcstein) [inline docs/developers/node-tutorial.mdx:494]:
```suggestion
System version: amd64/darwin
```

File: docs/nodes/light-node.mdx [hunk 2]
```diff
@@ -473,10 +473,10 @@ version` command:
...
 ```output
 celestia version
-Semantic version: v0.4.2
-Commit: 75b122f736d0db758ef840a4bf3b2e6635e823bd
-Build Date: Thu Oct 20 22:48:39 EDT 2022
-System version: arm64/darwin
+Semantic version: v0.5.0-rc5
+Commit: 039facd94104c741f8fa995e80f1321776ae4ad3
+Build Date: Tue Nov 15 16:23:30 EST 2022
+System version: amd64/darwin
```
Comment (jcstein) [inline docs/nodes/light-node.mdx:479]:
```suggestion
System version: arm64/darwin
```

File: docs/nodes/light-node.mdx [hunk 3]
```diff
@@ -503,10 +503,10 @@ version` command:
...
 ```output
 celestia version
-Semantic version: v0.4.2
-Commit: 75b122f736d0db758ef840a4bf3b2e6635e823bd
-Build Date: Thu Oct 20 22:48:39 EDT 2022
-System version: amd64/darwin
+Semantic version: v0.5.0-rc5
+Commit: 039facd94104c741f8fa995e80f1321776ae4ad3
+Build Date: Tue Nov 15 16:23:30 EST 2022
+System version: arm64/darwin
```
Comment (jcstein) [inline docs/nodes/light-node.mdx:509]:
```suggestion
System version: amd64/darwin
```

**Review by jcstein — COMMENTED** [review]
added small fixes to return values to avoid future tiny PRs 😄

**Review by jcstein — APPROVED** [review]
lgtm now

File: docs/developers/celestia-node-metrics.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ command:
 
 <!-- markdownlint-disable MD013 -->
 ```sh
-celestia light start --core.ip <ip-address> --core.grpc.port <port> --metrics --metrics.endpoint <ip-address:port>
+celestia light start --core.ip <ip-address> --core.grpc.port <port> --metrics --metrics.endpoint <ip-address:port> --gateway --gateway.ip <ip-address> --gatway.port <port>
```
Comment (Bidon15) [inline docs/developers/celestia-node-metrics.md:22]: please edit to `--gateway.addr` instead of `--gateway.ip` 🙏

Comment (jcstein) [reply docs/developers/celestia-node-metrics.md:22]: yes - adding to #317

**Pull request merged**